neurons/validator.py

```diff
@@ -76,6 +76,8 @@ def __init__(self):
         # === Bittensor objects ====
         self.wallet = bt.wallet(config=self.config)
         self.subtensor = bt.subtensor(config=self.config)
+        # Secondary archive subtensor to enable reading extrinsics of older blocks.
+        self.archive_subtensor = bt.subtensor(network=self.config.archive_endpoint)
```

Comment:
Does this need to be passed to the chain metadata store?

Reply:
Sure does! Great catch.